README.md

```diff
@@ -511,6 +511,27 @@ These settings can be changed at any time.
 * `settings.ecs.image-pull-behavior`: The behavior used to customize the [pull image process](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html#ecs-agent-availparam) for your container instances.
   Supported values are `default`, `always`, `once`, `prefer-cached`, and the default is `default`.
 
+#### Container runtime settings
+
+Some behavior of the container runtime (currently `containerd`) can be modified with the following settings:
+
+* `settings.container-runtime.enable-unprivileged-icmp`: Allow unprivileged containers to open ICMP echo sockets.
+* `settings.container-runtime.enable-unprivileged-ports`: Allow unprivileged containers to bind to ports < 1024.
+* `settings.container-runtime.max-concurrent-downloads`: Restricts the number of concurrent downloads for each image.
```

Comment:
For my own knowledge - concurrent downloads means "how many layers for a given image are downloaded at once", rather than "how many images are downloaded at once"?

I assume this does not actually allow `containerd` to download the same image multiple times, so it might be worth clarifying that it's about layers and manifests somehow (if that's true).

Reply:
Tweaked.